avm/ptn/alz/empty/main.bicep

```diff
@@ -0,0 +1,487 @@
+metadata name = 'avm/ptn/alz/empty'
+metadata description = '''
+Azure Landing Zones - Bicep - Empty
+
+Please review the [Usage examples](#Usage-examples) below, but please ensure for the `max` example you review the entire [directory](tests/e2e/max) to see the supplementary files that are required for the example.
```

Comment:
```suggestion
Please review the [Usage examples](#Usage-examples) section in the module's readme, but please ensure for the `max` example you review the entire [directory](tests/e2e/max) to see the supplementary files that are required for the example.
```

Reply:
We should consider somebody reviewing the template as is (or e.g. from the Registry when navigating to the 'show source' feature)

Reply:
very fair have updated